### ISSUE TITLE
DOMAINS AND IP WORKING at the same time on nodes

### ISSUE THREAD

**Issue by mrbusysky** [thenewboston-developers/Project-Proposals#355]
# Project Details

#### How would you describe your project?
DOMAINS AND IP WORKING at the same time on node without breaking confirmations

#### What problem does this project solve?
Right now if people set up domains and SSL on any node, PV/CV, or BANK it will break their confirmations. My fix allows them to use domains and also allows them to confirm transactions. The reason confirmations break is because nodes currently talk to each other on an IP level. This allows them to accept transactions view secured domain, but talk to each other on an IP level.
This fix is on config file-level and requires no changes to the chain itself.

#### Who is the target market?
the chain to fix security issues

#### How will this project benefit the network and/or the community?
Fix user security issues that leak their IP address with connection to a specific wallet.

#### How is this project centered around thenewboston?
helps with security

#### What is the estimated completion date for this project? (rough estimates are acceptable)
Will take a lot of hours.

#### Please include a detailed roadmap for your project below. This must include a list of all milestones where payments will be awarded upon completion of each.
I will find new Nginx settings that will allow nodes to use ip's and secure domains at the same time. 
I will also provide a guide of how users can set up quick and easy domains and SSL that way the team can update the website's guide.

#### What is your Discord username?
MrSky#5242

#### How long do you estimate for the MVP to complete?
Lots of hours

#### Project Wallet
fe06754d2dbf23509e975809e9023b620509c1e6618472f16c9659f59e940337

**Comment by Kycermann** [comment]
We're not happy to have IP addresses with HTTPS in the wallet- but we are happy to have domains over HTTPS.

**Comment by mrbusysky** [comment]
> We're not happy to have IP addresses with HTTPS in the wallet- but we are happy to have domains over HTTPS.

This update is actually that I just called it HTTPS instead of domains. Currently, Keysign's banks can do this.

**Comment by Kycermann** [comment]
> > We're not happy to have IP addresses with HTTPS in the wallet- but we are happy to have domains over HTTPS.
> 
> This update is actually that I just called it HTTPS instead of domains. Currently, Keysign's banks can do this.

To clarify, this proposal is to integrate https://bank.example.com instead of http://1.2.3.4:80 into the wallet- and not https://1.2.3.4?

**Comment by mrbusysky** [comment]
> > > We're not happy to have IP addresses with HTTPS in the wallet- but we are happy to have domains over HTTPS.
> > 
> > 
> > This update is actually that I just called it HTTPS instead of domains. Currently, Keysign's banks can do this.
> 
> To clarify, this proposal is to integrate https://bank.example.com instead of http://1.2.3.4:80 into the wallet- and not https://1.2.3.4?

Correct, this is to fix a config file in the banks and pv/cv that will allow banks to use both ip and domains. Without changing the main chains code. This change is require for the new manger update to work good with nodes

**Comment by mrbusysky** [comment]
I will be providing a guide how to setup your node so you can use both domain and ip. With no big updates required to the main chain code.

**Comment by tspearing** [comment]
This will require changes to the validator and bank source code and we don't have the resources to do this as the alpha network is in maintenance mode.
